addons/website_blog/views/website_blog_templates.xml

```diff
@@ -25,549 +25,562 @@
-                    <t t-set="count" t-value="0"/>
-                    <t t-foreach="posts" t-as="post">
-                      <div class="col-lg-4">
-                        <h4>
-                            <a t-attf-href="#{blog_url('', ['blog', 'post'], blog=post.blog_id, post=post)}" t-field="post.name"></a>
-                            <span t-if="not post.website_published" class="text-warning">
-                                <span class="fa fa-exclamation-triangle ml8" title="Unpublished" role="img" aria-label="Unpublished"/>
-                            </span>
-                        </h4>
-                        <div class="text-muted">
-                            <span class="fa fa-calendar mr8" role="img" aria-label="Post date" title="Post date" />
-                            <span t-field="post.post_date"/>
-                            <span class="fa fa-folder-open" /> In
-                            <a class="mr8" t-attf-href="#{blog_url(blog=post.blog_id)}">
-                            <span t-field="post.blog_id"/>
-                            </a>
-                        </div>
-                        <div class="text-muted">
-                            <span t-if="editable or post.tag_ids" class="fa fa-tags" role="img" aria-label="Tags" title="Tags"/>
-                            <t t-foreach="post.tag_ids" t-as="one_tag">
-                              <a class="badge badge-primary mr8" t-attf-href="#{blog_url('', ['blog', 'post'], blog=post.blog_id, tag=one_tag.id)}" t-esc="one_tag.name"/>
-                            </t>
-                            <span t-if="editable and not post.tag_ids" class="badge badge-danger">
-                                No tags defined!
-                            </span>
-                        </div>
-                        <div class="text-muted" t-if="len(post.message_ids) &gt; 0">
-                            <span class="fa fa-comment-o"/>
-                            <a t-attf-href="#{blog_url('', ['blogpost'], blogpost=post)}#comments">
-                                <t t-if="len(post.message_ids) &lt;= 1" ><t t-esc="len(post.message_ids)"/> comment</t>
-                                <t t-if="len(post.message_ids) > 1"><t t-esc="len(post.message_ids)"/> comments</t>
-                            </a>
-                        </div>
-                        <div class="text-muted mb16" t-field="post.website_meta_description"/>
-                      </div>
-                      <t t-set="count" t-value="count+1"/>
-                      <div class="clearfix" t-if="(count % 3) == 0"/>
-                    </t>
-                </div>
-            </section>
-            <div class="oe_structure" id="oe_structure_website_blog_latest_blogs_2"/>
+        <div id="wrap" class="js_blog website_blog">
+            <t t-raw="0"/>
+
+            <!-- Droppable-area shared across all blog's pages -->
+            <div class="oe_structure oe_empty o_wblog_structure_shared"
```

Comment:
> When you create an oe_structure you now have to add an ID to it which begins by oe_structure_

I was not aware. I sow the IDs but I thought were used to xpath (and since these xpaths were not needed anymore I removed the ID).
Thanks

Reply:
You have right ;) it is really for xpath... not existing... but future.

When you add an id, if a user drop a snippet at this place, that will automatically create a new view with an xpath on this id. From this way, the original view is not updated, and during the next migration, he will have your new design and new feature.